src/js/releases.js

```diff
@@ -94,7 +96,7 @@ function populateLatest() {
 
         // set the download links for this release...
 
-        if(a.indexOf("LINUX") >= 0) { // if the binary name contains 'LINUX':
+        if(a.indexOf("X64_LINUX") >= 0) { // if the binary name contains 'LINUX':
```

Comment:
This check may need tweaking again in the near future when we fix the binaries to have the correct CPU architecture identifiers.